packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title styles="color:white">Welcome to .NET Core!</title>
+</head>
+<body>
+    <font face="Helvetica">
+        <h3>
+            Welcome to .NET Core!
+        </h3>
+        <p>
+            Learn more about .NET Core at <a href="https://aka.ms/dotnet-docs">https://aka.ms/dotnet-docs</a> . Use dotnet --help to see available commands or go to <a href="https://aka.ms/dotnet-cli-docs">https://aka.ms/dotnet-cli-docs</a>.
+        </p>
+        <h3>
+            Telemetry
+        </h3>
+         <p>
+            The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community. You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell. You can read more about .NET Core tools telemetry at <a href="https://aka.ms/dotnet-cli-telemetry">https://aka.ms/dotnet-cli-telemetry</a>.
+        </p>
+        <h3>
+            Configuration
+        </h3>
+        <p>
+            A command is run in the end of install process to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.
```

Comment:
It is odd, since I moved it to after installation. I am thinking to send a different pull request to fix and unify the text across. I am adding the deb version currently, and there might be more change to the text.

#6448